modules/webbrowser/src/webbrowsermodule.cpp

```diff
@@ -112,6 +113,14 @@ WebBrowserModule::WebBrowserModule(InviwoApplication* app)
     // Call 60 times per second
     , doChromiumWork_(Timer::Milliseconds(1000 / 60), []() { CefDoMessageLoopWork(); }) {
```

Comment:
Don’t you need to use the WebbrowserSettings default value here?

Reply:
sure, forgot about to change it here.